src/DDTrace/Type.php

```diff
@@ -4,6 +4,7 @@
 
 class Type
 {
+    const FILESYSTEM = 'filesystem';
```

Comment:
Given that these functions can do more than filesystems, I think we should not name it 'filesystem'. I've been thinking more about it; these are really stream operations. Maybe we call it 'phpstream' or something?